gnovm/pkg/gnomod/pkg.go

```diff
@@ -5,40 +5,32 @@ import (
 	"io/fs"
 	"os"
 	"path/filepath"
+	"strings"
 )
 
 type Pkg struct {
-	name     string
-	path     string
-	draft    bool
-	requires []string
+	Dir      string   // absolute path to package dir
+	Name     string   // package name
+	Requires []string // dependencies
+	Draft    bool     // whether the package is a draft
+}
+
+type SubPkg struct {
+	Dir        string   // absolute path to package dir
+	ImportPath string   // import path of package
+	Root       string   // Root dir containing this package, i.e dir containing gno.mod file
+	Imports    []string // imports used by this package
+
+	GnoFiles         []string // .gno source files (excluding TestGnoFiles, FiletestGnoFiles)
+	TestGnoFiles     []string // _test.gno source files
+	FiletestGnoFiles []string // _filetest.gno source files
```

Comment:
maybe
FilePaths []string
TestFilePaths []string
FiletestFilePaths []string

BTW we might support other langauges that compile to .gno, like .gnoffee

Reply:
The original thought process behind this is to have pkg in a structured way.

If we want to support another language we can add another field here. 
e.g. `PyFiles []string` `GnoffeeFiles []string`...

This way we can apply specific operations on them accordingly. 
e.g. 
`GnoffeeFiles []string` - needs to be compiled to .gno
`GnoFiles []string` - vm understands it natively 
`TestGnoFiles []string` - if you wants to run tests, these are your files